debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SpecialCharsInNamesIT.java

```diff
@@ -48,6 +49,75 @@ public void after() throws SQLException {
         }
     }
 
+    @Test
+    @FixFor("DBZ-1546")
```

Comment:
I like that you've chosen this test class. But could you please use the test schema/table from other method that has space inside its name?

Reply:
👍